doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -91,68 +84,84 @@ and tables:
 Flags, options and tables are identified by a series of identifiers, each with an initial
 capital letter.
 
-.. cmd::  Set @flag
+.. cmd:: {? Export } Set {+ @ident }
    :name: Set
 
-   Sets :token:`flag` on.
+   Sets the flag identified by the :n:`@ident`\s to on.
 
-.. cmd:: Unset @flag
-   :name: Unset
+   This command supports the :attr:`local`, :attr:`global` and :attr:`export` attributes.
+   They are described :ref:`here <set_unset_scope_qualifiers>`.
 
-   Sets :token:`flag` off.
+   .. todo shouldn't export be included in the legacy_attr NT and listed in the table
+      in the gallina/Attributes section?  And not shown here?
 
-.. cmd:: Test @flag
+   .. todo "Set Hyps Limit." gives "Bad type of value for this option: expected int, got bool."
+      I can create an issue--when do you think someone might get to this?  The issue backlog
+      is another topic for discussion/action.
 
-   Prints the current value of :token:`flag`.
+.. cmd:: {? Export } Set {+ @ident } {| @int | @string }
+   :name: Set @option
 
+   Sets the option identified by the :n:`@ident` to the specified value, which
+   must be of the appropriate type for the option.
 
-.. cmd:: Set @option {| @num | @string }
-   :name: Set @option
+   This command supports the :attr:`local`, :attr:`global` and :attr:`export` attributes.
+   They are described :ref:`here <set_unset_scope_qualifiers>`.
+
+.. cmd:: {? Export } Unset {+ @ident }
+   :name: Unset
 
-   Sets :token:`option` to the specified value.
+   If :n:`@ident`\s is a flag, it is set to off.  If :n:`@ident`\s is an option, it is
+   set to its default value.
 
-.. cmd:: Unset @option
-   :name: Unset @option
+   This command supports the :attr:`local`, :attr:`global` and :attr:`export` attributes.
+   They are described :ref:`here <set_unset_scope_qualifiers>`.
 
-   Sets :token:`option` to its default value.
+   .. todo doesn't work for "Diffs"
 
-.. cmd:: Test @option
+.. cmd:: Test {+ @ident }
+   :name: Test @flag/@option
 
-   Prints the current value of :token:`option`.
+   Prints the current value of the flag or option identified by the :n:`@ident`\s.
 
 .. cmd:: Print Options
 
    Prints the current value of all flags and options, and the names of all tables.
 
 
-.. cmd:: Add @table {| @string | @qualid }
+.. cmd:: Add @ident {? @ident } {+ {| @qualid | @string } }
    :name: Add @table
 
-   Adds the specified value to :token:`table`.
+   Adds the specified values to the table identified by the :n:`@ident`.
 
-.. cmd:: Remove @table {| @string | @qualid }
-   :name: Remove @table
+   .. todo: are tables identified by a single ident or is this command limited
+      What are the first two idents?
+      It handles multiple values?
+      Is it used much?
```

Comment:
All the tables that are currently documented are two ident long. My guess is that a parsing disambiguation problem prevents from having longer table names.